src/sql/store.py

```diff
@@ -130,6 +130,8 @@ def __str__(self) -> str:
         template = (
             with_clause_template_backtick if is_use_backtick else with_clause_template
         )
+        if len(with_all) == 0:
```

Comment:
add a comment here to explain why we need this if statement

Reply:
Added `# return query without 'with' when no dependency exists` above the if statement